package.json

```diff
@@ -38,6 +38,7 @@
     "test": "run-s test:circular lint format:check",
     "test:circular": "dpdm --warning false --tree false --exit-code circular:1 ./src/index.ts",
     "lint": "eslint . --color",
+    "lint-touched-file": "eslint --color --max-warnings=0",
```

Comment:
In general, I think it will be a good idea to decompose the yarn build script.
Such a change would allow running the circle-deps check and linter in separate steps, and also allow for lining only the files that changed in the PR instead of the entire code base!